### PR TITLE
Add BeepConfig command support, miscellaneous beep related cleanup

### PR DESCRIPTION
**Pull request by itsmojo**

### CODE REVIEW

File: OmniKit/MessageTransport/MessageBlocks/BeepConfigCommand.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//
+
+import Foundation
+
+public struct BeepConfigCommand : MessageBlock {
+    // https://github.com/openaps/openomni/wiki/Command-1E-Beep-Config
+
+    public let blockType: MessageBlockType = .beepConfig
+    public let length: UInt8 = 4
+    public let beepType: UInt8
```
Comment (ps2) [inline OmniKit/MessageTransport/MessageBlocks/BeepConfigCommand.swift]: UInt8 -> BeepType

**Review by ps2 — COMMENTED** [review]
Looks good!  Just one recommendation; MessageBlocks should use the decoded types as their members, and encode/decode when serializing or deserializing.

File: OmniKit/MessageTransport/MessageBlocks/BeepConfigCommand.swift (hunk 1 above)
Comment (itsmojo) [reply OmniKit/MessageTransport/MessageBlocks/BeepConfigCommand.swift]: Done

Comment (ps2) [reply OmniKit/MessageTransport/MessageBlocks/BeepConfigCommand.swift]: Thanks Joe!

**Pull request merged**